docs/high-level-overview.md

```diff
@@ -1,165 +0,0 @@
-# High-Level Bridge Documentation
```

Comment:
I am not sure if we should keep those docs, some of them are useful IMO.

Reply:
The deleted docs can still be found from upstream.

OK to delete them before rewriting.

Reply:
Make sense.

Reply:
the docs will be modified and added later.